pkg/reconciler/v1alpha1/revision/config/observability.go

```diff
@@ -48,6 +48,9 @@ type Observability struct {
 	// LoggingURLTemplate is a string containing the logging url template where
 	// the variable REVISION_UID will be replaced with the created revision's UID.
 	LoggingURLTemplate string
+
+	// RequestLogTemplate is the go template to use to shape the request logs.
+	RequestLogTemplate string
```

Comment:
see below, store as a Go template.

Reply:
I need to store this as a string, because we will pass this to the queue proxy as an env variable. I don't see any way to get the string from the template object back.